utils/litemap/src/map.rs

```diff
@@ -35,6 +35,21 @@ impl<K, V> LiteMap<K, V> {
     }
 }
 
+impl<K, V, S> LiteMap<K, V, S> {
+    /// Construct a new [`LiteMap`] using the given values
+    ///
+    /// # Safety
+    ///
+    /// The store must be sorted and have no duplicate keys.
+    pub const fn from_store_unchecked(values: S) -> Self {
```

Comment:
We have `from_tuple_vec_unchecked` and `from_sorted_slice_unchecked`, one with `_sorted`, one without (we can actually replace those two by `from_store_unchecked`). I'm ambiguous about `_sorted`, `sorted_store` seems a bit redundant because `Store`s are always sorted, but the underlying types aren't so maybe add it.

Reply:
Naming it `from_sorted_store_unchecked`.